tests/test_registrations/test_models.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+'''Unit tests for models and their factories.'''
+from nose.tools import *  # noqa (PEP8 asserts)
+import mock
+import unittest
+
+from website.models import MetaSchema, DraftRegistrationApproval
+
+from tests.factories import (
+    UserFactory, ApiOAuth2ApplicationFactory, NodeFactory, PointerFactory,
+    ProjectFactory, NodeLogFactory, WatchConfigFactory,
+    NodeWikiFactory, RegistrationFactory, UnregUserFactory,
+    ProjectWithAddonFactory, UnconfirmedUserFactory, CommentFactory, PrivateLinkFactory,
+    AuthUserFactory, DashboardFactory, FolderFactory,
+    NodeLicenseRecordFactory, DraftRegistrationFactory
+)
+from tests.test_registrations.base import RegistrationsTestBase
+
+
+class TestDraftRegistrations(RegistrationsTestBase):
+
+    def test_factory(self):
+        draft = DraftRegistrationFactory()
+        assert_is_not_none(draft.branched_from)
+        assert_is_not_none(draft.initiator)
+        assert_is_not_none(draft.registration_schema)
+
+        user = AuthUserFactory()
+        draft = DraftRegistrationFactory(initiator=user)
+        assert_equal(draft.initiator, user)
+
+        node = ProjectFactory()
+        draft = DraftRegistrationFactory(branched_from=node)
+        assert_equal(draft.branched_from, node)
+        assert_equal(draft.initiator, node.creator)
+
+        schema = MetaSchema.find()[1]
```

Comment:
Is there any reason to expect that the content and number of metaschemas will be consistent? (eg, is `Metaschema.find()[1]` always going to get the schema you intend?)